ar_exercise/exercise.rb

```diff
@@ -32,17 +32,38 @@ class Apartment < ActiveRecord::Base
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
 all_tenants = Tenant.all
-
 # get the first tenant in the DB
+first_tenants = Tenant.first
 # get all tenants older than 65
+old_tenants = Tenant.find_by(age: >65)
 # get all apartments whose price is greater than $2300
+expensive_apartements = apartments.find_by(monthly_rent: >2300)
 # get the apartment with the address "6005 Damien Corners"
+specfic_apartement = apartments where adress= "6005 Damien Corners"
 # get all tenants in that apartment
+specfic_apartement=gets.chomb
+tenane_in_apartement=Tenant.find_by(apartment_id:specfic_apartement)
 
 # Use `each` and `puts` to:
+
 # Display the name and ID # of every tenant
+all_tenants=Tenant.all
+name=all_tenants.name
+id=all_tenants.id
```

Comment:
You cannot use `.id` on `Tenant.all` since that contains multiple tenants. You can only call `.id` on a specific instance of the class.